index.bs

```diff
@@ -1149,9 +1203,68 @@ The [=new=] {{MLGraphBuilder}} constructor steps are:
 1. If [=this=]'s [=relevant global object=]'s [=associated Document=] is not [=allowed to use=] the [=webnn-feature|webnn=] feature, throw a "{{SecurityError}}" {{DOMException}} and abort these steps.
 1. Let |context| be the first argument.
 1. If the <a>validate MLContext</a> steps given |context| return `false`, throw a "{{TypeError}}" and abort these steps.
-
 1. Set {{MLGraphBuilder/[[context]]}} to |context|.
 
+### The `constant()` method  ### {#api-mlgraphbuilder-constant-method}
```

Comment:
I'm a bit confused by the styling of the section name. For example, here you have the `constant()` method name of section 7.6.2 as a quoted string, which produces a monospaced display text in the content, while all the existing sections e.g. the subsequent one 7.6.3 having the batchNormalization name outside of the quote. But then the sections before it have a mixture of quoted and unquoted method names, etc. 

This is the reason behind calling out making a stylistic change a separate PR in our [Contribution Guidelines](https://github.com/webmachinelearning/webnn/blob/main/CONTRIBUTING.md#type-of-change). We have a lot of content in the spec, if a PR comes in with both content and styling changes, we may have missed something during the reviews and caused the document to become inconsistent styling-wise. 

My recommendation for this PR is to separate out styling changes into a separate PR as described in the guidelines.

Reply:
Thanks for taking a look.

If you check this thread, earlier we discussed why the back quotes are used. The `constant()` method is polymorphic. It has 2 forms like `{{MLGraphBuilder/constant(descriptor, bufferView)}}` and `{{MLGraphBuilder/constant(value, type)}}`.
Since neither is used in the title of the main section, the simple back quoted `constant` is used there. This is consistent over my subsequent changes.

For other methods the autolinking definition should be used. 

To apply first the stylistic changes, means a lot of unwelcome manual merge for me. In addition, some of the sections do not even exist yet to which the stylistic changes will be applied. So we'd apply first those stylistic changes that apply to the current version of the document, then those which apply to the additional sections. So anyway we can't avoid doing later stylistic changes, but we can make it extremely complicated for the rules' sake.

Reply:
The reason we agreed to do a few PRs first was exactly to figure out consistent stylistic changes across these selected features, which will then be applied to all subsequent PRs. I mentioned in the last call that this makes the currently open PRs a bit of an exception. We need them to figure out the rules for the single-shot stylistic PR you've been asking for.

Otherwise we can't avoid going back and forth - which is okay for a few exceptions but we want to get it right before scaling up. 

So if the batchNorm PR differs, that is its bug, and I will correct that. By the time we merge these PRs we will have the rules pretty much agreed upon. There are not so many of them, anyway.

Reply:
I have removed the back quotes from the title and made a note to self to record that preference.